### ISSUE TITLE
Error 500 on "/user/logout"

### ISSUE THREAD

**Issue by abs0lom** [SIGIRE/CoRM#16]
On a fresh install of CoRM, the logout url launch a 500 error.

I found error on production.log : `NoMethodError (undefined method 'to_datetime' for nil:NilClass)`

Any suggestions ?

**Comment by SIGIRE** [comment]
Could you please specify the ruby version and the OS you are using and post the end of the production.log file?

**Comment by abs0lom** [comment]
Ruby version : 2.3.3p222
OS : Debian 9 Stretch
PostgreSQL : 9.6.9
end of production.log in attached file
[production.log](https://github.com/SIGIRE/CoRM/files/2232773/production.log)

Thank you :)

**Comment by SIGIRE** [comment]
The first line of your production.log file shows a logout action.
Could you specify what action you made before the error ? Are you able to repeat the problem ?

**Comment by abs0lom** [comment]
Yes, the bug appear every time when I go to the "/user/logout" url.
This action don't logout me : if I load another page, I'm already logged and I can repeat the logout action to see again the problem.

**Comment by SIGIRE** [comment]
Please, post your gemfile file.

**Comment by abs0lom** [comment]
[Gemfile.lock](https://github.com/SIGIRE/CoRM/files/2260236/Gemfile.lock.txt)
[Gemfile](https://github.com/SIGIRE/CoRM/files/2260237/Gemfile.txt)
